testing/src/main/java/org/oppia/android/testing/network/MockRetrofitHelper.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.android.testing.network
+
+import okhttp3.OkHttpClient
+import org.oppia.android.data.backends.gae.NetworkInterceptor
+import org.oppia.android.data.backends.gae.NetworkSettings
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.mock.MockRetrofit
+import retrofit2.mock.NetworkBehavior
+import javax.inject.Inject
+
+/**
+ * This helper allows us to create customized [MockRetrofit] instances.
+ */
+class MockRetrofitHelper @Inject constructor() {
```

Comment:
We're constructing this elsewhere rather than injecting it. Could we inject it, instead?

That being said, could we maybe make this a test module & make ``MockRetrofit`` injectable? That might help reduce potential user error in the future since it forces the one ``MockRetrofit`` instance in the Dagger graph to be a correctly configured one.

Reply:
Done.